src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Figure implements Area {
```

Comment:
Create an interface AreaCalculator and implement it here, it should not be done with Drawable and draw(). This class should be abstract - we never instantiate it directly, only extend